   syntax = "proto3";

   message NotificationTupeMoc
   {
     uint32 typeNotification=1;
   }